web/navigation-v3/index.html

```diff
@@ -183,67 +206,85 @@ <h2 class="visuallyhidden">Main</h2>
 								<a href="page.html" class="nav-link">Business Ecosystems</a>
 								<ul class="clean-list" role="list">
 										<li>
-											<a href="page.html" class="nav-link">Automotive and transportation</a>
+											<a href="page.html" class="nav-link">eCommerce</a>
 										</li>
 										<li>
 											<a href="page.html" class="nav-link">Media and entertainment</a>
 										</li>
+										<li>
+											<a href="page.html" class="nav-link">Network and communications</a>
+										</li>
 										<li>
 											<a href="page.html" class="nav-link">Publishing</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Web payments</a>
+											<a href="page.html" class="nav-link">Security and privacy</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Web of Data</a>
+											<a href="page.html" class="nav-link">Smart cities</a>
+										</li>
+										<li>
+											<a href="page.html" class="nav-link">Transportation</a>
+										</li>
+										<li>
+											<a href="page.html" class="nav-link">Web advertising</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Web and telecommunications</a>
+											<a href="page.html" class="nav-link">Web of Data</a>
 										</li>
 										<li>
 											<a href="page.html" class="nav-link">Web of Things</a>
 										</li>
 									</ul>
 							</li>
+							<li>
+								<a href="page.html" class="nav-link">Liaisons</a>
+							</li>
 							<li class="has-children">
-								<a href="page.html" class="nav-link">Supporting W3C</a>
+								<a href="page.html" class="nav-link">Support W3C</a>
 								<ul class="clean-list" role="list">
 										<li>
-											<a href="page.html" class="nav-link">Donate now</a>
+											<a href="page.html" class="nav-link">Donate</a>
 										</li>
 										<li>
-											<a href="page.html" class="nav-link">Global sponsorship</a>
-										</li>
-										<li>
-											<a href="page.html" class="nav-link">More ways to support us</a>
+											<a href="page.html" class="nav-link">Sponsorship</a>
 										</li>
 									</ul>
 							</li>
```

Comment:
To match the current sitemap "Donate" and "Sponsoring W3C" (rather than "Sponsorship" par Coarlie's preference) should be directly under "Get involved" (as we won't have an umbrella "Support W3C" page).

Reply:
OK, I thought the orange lines indicated those lines were marked for deletion but it seems the opposite is the case. If this is something that has been decided, can't we simply update the google doc?